packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -491,7 +523,7 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
             expanded: true,
             id: rootUri,
             parent: this.model.root as SearchInWorkspaceRoot,
-            visible: this.workspaceService.isMultiRootWorkspaceOpened
+            visible: this.forceVisibleRootNode || this.workspaceService.isMultiRootWorkspaceOpened
```

Comment:
I think we can simplify the condition to `this.resultTree.size > 1` to align with VS Code.

Right now it displays the root node while in a multi-root workspace even if we find results under a single root, VS Code does not.

Reply:
Just tried and apparently it doesn't work like this alone.